tests/test_apidoc.py

```diff
@@ -118,6 +120,15 @@ def test_pep_0420_disabled_top_level_verify(make_app, apidoc):
     print(app._status.getvalue())
     print(app._warning.getvalue())
 
+@pytest.mark.apidoc(coderoot=(rootdir / 'root' / 'trailing_underscore'))
```

Comment:
Actually, I need 'text' builds, since otherwise it's not a good test. I've moved the files, and updated the tests